include/mata/nfa.hh

```diff
@@ -952,99 +772,50 @@ void concatenate(Nfa* res, const Nfa& lhs, const Nfa& rhs, bool use_epsilon = fa
  * @param[out] rhs_result_states_map Map mapping rhs states to result states.
  * @return Concatenated automaton.
  */
+        // TODO: check how fast is using just concatenate over epsilon and then call remove_epsilon()
 Nfa concatenate(const Nfa& lhs, const Nfa& rhs, bool use_epsilon = false,
                 StateToStateMap* lhs_result_states_map = nullptr, StateToStateMap* rhs_result_states_map = nullptr);
 
 /// makes the transition relation complete
+// TODO: complete().
 void make_complete(
```

Comment:
Why did we not rename this function to `complete()` as suggested?

Reply:
Actually, it sounds like predaticate to me. `bool complete()`. I wanted to discuss this. I think `make_complete` explains better semantics of function.

Reply:
yes, makes sense

Reply:
I removed the TODO

Reply:
I wholeheartedly agree with you. Thanks for removing the TODO.